src/store/modules/chat/index.js

```diff
@@ -564,7 +564,7 @@ const actions = {
             {
               partnerId: contactId,
               commands: messages
-                .filter((item) => item.message.startsWith('/'))
+                .filter((item) => item.message?.startsWith('/'))
```

Comment:
Verify using `typeof string`, because in case of crypto transfer, the `message` is an object type

![image](https://github.com/Adamant-im/adamant-im/assets/25831507/4e5e9ea4-b3bd-4a37-882b-821cede29c5d)